src/components/organisms/EarthEditor/CanvasArea/convert.ts

```diff
@@ -145,7 +145,7 @@ type GQLLayer = Maybe<
     EarthLayerCommonFragment
 >;
 
-const processLayer = (layer: Maybe<GQLLayer>): Layer | undefined => {
+export const processLayer = (layer: Maybe<GQLLayer>): Layer | undefined => {
```

Comment:
I left it because i was not sure if the whole plan (move layerStore creation to molecules) is okey and if not i still can change it back quickly. I'm deleting it now.